test/unit/spec/PrimaryAuth_spec.js

```diff
@@ -315,37 +315,43 @@ function (Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthForm, Be
           expect(explain.length).toBe(0);
         });
       });
-      itp('username field does have default explain when label is customized', function () {
+      itp('username field does not have explain when only label is customized', function () {
         var options = {
+          'language': 'en',
           'i18n': {
             'en': {
-              'primaryauth.username.placeholder': 'Custom Username Explain'
+              'primaryauth.username.placeholder': 'Custom Username Label'
             }
+          },
+          'features' : {
+            'hideDefaultTip': true
```

Comment:
nit: Do we need this? It is set to true by default right?

Reply:
Good point! Updated, thank you!